docs/aineko_dream.md

```diff
@@ -0,0 +1,26 @@
+# Aineko Dream
+
+Aineko Dream is the quickest way to get started with Aineko pipelines by leveraging the power of generative AI to create a starter Aineko pipeline based on your use case. 
```

Comment:
I would make this more concise and use the active voice. Using Aineko 3 times sounds too repetitive imo.

```suggestion
Aineko Dream leverages the power of generative AI to create a starter Aineko pipeline based on your use case. 
```